README.md

```diff
@@ -34,14 +34,14 @@ const settings = {
 export default settings;
 ```
 
-⚠️  Please do not present API credentials in public places.
+⚠️  Please do not display/use API credentials in public places/open WIFI.
 
 ### Dependencies
 
-Before you start working with Infermedica API and JS ES6, please make sure that you have `node` installed in your system.
+Before you start working with Infermedica API and JS ES6, please ensure that you have `node` installed in your system.
```

Comment:
I would still say: Before you start working with this example, please ... Because node isn't required for using the API in general.